src/richie/apps/search/indexers/persons.py

```diff
@@ -96,6 +101,7 @@ def get_es_document_for_person(cls, person, index=None, action="index"):
             },
             "portrait": portrait_images,
             "title": titles,
+            "title_raw": titles,
```

Comment:
Can you explain why we can't sort on "titles"?

Reply:
I added a comment in the relevant mappings. The analysis alters the values and prevents us from using them for sorting (ES raises an exception, but I think the ordering wouldn't make any sense even if it did not).

Reply:
IC :pray: